site/en/docs/privacy-sandbox/fledge-best-practices/index.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'FLEDGE latency best practices'
+subhead: >
+  How to get the best performance from FLEDGE usage
+description: >
+  Collection of best practices to improve FLEDGE auction latency
+date: 2022-06-27
+authors:
+  - pauljensen
+---
+
+<!--lint disable no-smart-quotes-->
+
+It's in everyone's best interest to make sure FLEDGE operates efficiently.
+
+* People browsing the web want sites to load quickly. This means FLEDGE must not overutilize limited device resources, like compute or network resources, necessary to load sites and their embedded ads.
+* Publishers want their sites to load quickly, providing users an efficient and responsive experience. Publishers also want effective advertising to maximize their revenue.
+* Advertisers and adtechs want their ads to display quickly to provide the greatest utility.
+
+This document seeks to outline some behaviors to ensure FLEDGE operates at maximum efficiency.
+
+
+## Buyer (bidder) best practices
+
+To make sure you're optimizing for FLEDGE auction efficiency, follow these best practices
+
+### Fewer interest group owners
+
+To protect FLEDGE bidders in the same way that the browser protects different origins on the web using [site isolation](https://www.chromium.org/Home/chromium-security/site-isolation/), the browser uses expensive resources (like operating system processes) to protect individual interest group owners.
+
+To minimize the expenditure of these very expensive resources, having the fewest number of interest group owners is crucial. Avoid having different interest groups owned by various subdomains, for example if `adtech.example` had interest groups owned by cats.`adtech.example` and `dogs.adtech.example`, the browser would likely use two separate processes to run their bidding scripts.
+
+### Fewer interest groups bidding
+
+The browser must do significant setup and preparation before invoking a buyer's `generateBid()` script, such as setting up a new clean JavaScript execution environment, and parsing and loading the `generateBid()` code.
```

Comment:
I'm confused by what "do significant setup and preparation" means. Can you clarify?

Reply:
I think in this case, the second sentence "such as setting up a new clean JavaScript execution environment, and parsing and loading the `generateBid()` code" describes that setup/preparation part.  If you think it needs more clarification, it may be worth reaching out to the original author of the article.